components/record/Metadata.vue

```diff
@@ -6,19 +6,24 @@
       :data-qa="'field ' + key"
       class="border-bottom mb-3"
     >
-      <div><strong>{{ key }}</strong></div>
+      <div><strong>{{ fieldLabel(key) }}</strong></div>
```

Comment:
Use a [Vue computed property](https://vuejs.org/v2/guide/computed.html) instead of calling the function here.

Reply:
That likely however implies moving the template for one metadata field into another component with a computed property for its key.

Reply:
Or perhaps more simply just moving the `v-for` up to the parent component and making this component responsible for rendering a single metadata field, renaming it to `MetadataField.vue`.

Reply:
You can create a custom (global) filter as well for this renaming. 

https://vuejs.org/v2/guide/filters.html
https://scotch.io/tutorials/how-to-create-filters-in-vuejs-with-examples